selfdrive/car/toyota/carcontroller.py

```diff
@@ -159,6 +157,10 @@ def update(self, sendcan, enabled, CS, frame, actuators,
       apply_steer_req = 0
     else:
       apply_steer_req = 1
+    
+    apply_steer = apply_toyota_steer_torque_limits(apply_steer, self.last_steer, CS.steer_torque_motor, SteerLimitParams)
+    if apply_steer == 0 and self.last_steer == 0:
+      apply_steer_req = 0
 
```

Comment:
you don't want panda to block messages after a user cancellation. 
I'm fine with this change but the `if not enable the apply_torque = 0` shoudl stay after `apply_toyota_steer_torque_limits`.

Makes sense?